rcon/auto_settings.py

```diff
@@ -181,18 +182,30 @@ def create_condition(name, **kwargs):
         raise ValueError("Invalid condition type: %s" % name)
 
 
-def do_run_commands(rcon, commands):
+def do_run_commands(rcon, votemap, commands):
     for command, params in commands.items():
         try:
             logger.info("Applying %s %s", command, params)
-            rcon.__getattribute__(command)(**params)
+            vm_commands = {
+                "do_add_map_to_whitelist": votemap.do_add_map_to_whitelist,
+                "do_add_maps_to_whitelist": votemap.do_add_maps_to_whitelist,
+                "do_remove_map_from_whitelist": votemap.do_remove_map_from_whitelist,
+                "do_remove_maps_from_whitelist": votemap.do_remove_maps_from_whitelist,
+                "do_reset_map_whitelist": votemap.do_reset_map_whitelist,
+                "do_set_map_whitelist": votemap.do_set_map_whitelist,
```

Comment:
Oh wow you went all the way. 
I was really just thinking of a list in the config.yml and we call it a day 😅

Reply:
Now we have to add a UI

Reply:
If we have set_whitelist(maos) I can just reuse the UI from the maprotation I think